may hack updates (#557)
Consensys/doc.linea: docs/build-on-linea/hackathons.md

- Non-trivial progress made on your project during that month (as measured by your Github commit history)
- Verified smart contract deployed on Linea Sepolia or Linea Mainnet
- Working product demo link

> - Short video explainer and demo (less than 5 minutes) of of your project and, if it's not a new project, what changes you made that month

Reviewer: ```suggestion
- Short video explainer and demo (less than 5 minutes) of your project and, if it's not a new project, what changes you made that month
```

#### How to submit

To submit to the May hackathon, just follow the [Linea builder launchpad steps](https://aspecta.id/builder-matrix/Linea-builder-launchpad)! You'll be able to receive builder NFTs, see the skills of other builders to form teams, and submit your project there!

### May 2024 (1 ETH pool prize + $500 USD [Dune API](https://docs.dune.com/api-reference/overview/introduction) pool prize) :frog:

> This month, you can win both the 1 ETH pool prize in addition to extra prize money for using the [Dune API](https://docs.dune.com/api-reference/overview/introduction). We'll be prioritizing submissions that are building a short cut via [onthis.xyz](https://onthis.xyz) or with the Dune API.

Reviewer: ```suggestion
This month, you can win both the 1 ETH pool prize in addition to extra prize money for using the [Dune API](https://docs.dune.com/api-reference/overview/introduction). We'll be prioritizing submissions that are building a shortcut via [onthis.xyz](https://onthis.xyz) or with the Dune API.
```

**Register and submit through the [Linea Builder Launch Pad](https://aspecta.id/builder-matrix/Linea-builder-launchpad)**

**Due 11:59PM EST, May 31st, 2024**

#### 1 ETH pool prize

> We'll be distributing 1 ETH amongst the top qualified projects. This can take the form of 2 outstanding projects receiving 0.5 ETH each, 10 projects receiving 0.1 ETH each, or, in the case that no submissions meet our quality bar, no ETH being distributed for that month.

Reviewer: ```suggestion
We'll be distributing 1 ETH amongst the top qualified projects. This can take the form of two outstanding projects receiving 0.5 ETH each, 10 projects receiving 0.1 ETH each, or, in the case that no submissions meet our quality bar, no ETH being distributed for that month.
```

We'll also be hosting a workshop with them on May 9th!

#### $500 USD [Dune API](https://docs.dune.com/api-reference/overview/introduction) pool prize

[Dune](https://dune.com) is crypto's data platform. Teams and individuals alike use Dune to explore, query, access, and export blockchain data. Dune's community of data engineers, devs, researchers, and analysts use the platform to gain deeper insights into blockchain activity. Dune provides organized, decoded and human-readable blockchain data, across 30+ chains and 1.5M+ datasets.

Dune is looking for the most creative use cases of the Dune API. If you don't know where to start, building a Farcaster frame with the Dune API is a great place to start!

**Resources:**

- Coming soon: How to build in Linea with Dune API (repo and quickstart)
- [Documentation](https://docs.dune.com/home): The official documentation provides detailed guides and tutorials on using Dune Analytics.
- [API Docs](https://docs.dune.com/home): The API documentation provides detailed information on how to access blockchain data programmatically using Dune's API.

> - [YouTube](https://www.youtube.com/@dunecom) : The Dune YouTube channel features video tutorials and demos of the platform.

Reviewer: ```suggestion
- [YouTube](https://www.youtube.com/@dunecom): The Dune YouTube channel features video tutorials and demos of the platform.
```
